### DISCUSSION TITLE
Any Code override tip to show menu button all the time?

### DISCUSSION THREAD

**Discussion by fishxom** [requarks/wiki#3047] [Help / Questions]
I want the option to hide the menu all the time, like when we reduce the size of the screen and the menu button shows-up, so you can show or hide it, even when maximized.

Looking to the code it seems simple, but I'm using the docker container and didn't want to do a fork of the code.

Is there a way I can achieve that by the Code Override in the Admin page? I tried with no success, but I don't fully understand that.

Thanks,

**Comment by aaronuu** [comment]
我也遇到同样的问题，希望可以有这个功能

**Reply by aaronuu** [reply]
希望可以有隐藏菜单这个功能
